templates/content/hall_of_fame.html.twig

```diff
@@ -2,8 +2,8 @@
 
 {% block seo %}
     {{ include('_seo.html.twig', {
-        title: 'Slack Secret Santa - Hall of Fame',
-        description: 'List of awesome companies using Slack Secret Santa',
+        title: 'Secret Santa for Teams - Hall of Fame',
```

Comment:
"Secret Santa for teams" avec un `t` miniscule ? Sinon on peut penser que vous parlez du logiciel de Microsoft :/